demos/capi/README.md

```diff
@@ -0,0 +1,89 @@
+# C and C++ API inference demo (C/C++)
+
+This demo creates a new docker image openvino/model_server-capi with two example applications that are using c or c++ ovms API.
+
+## Build ovms_shared.so
+To build the capi docker image, you must first build the ovms.tar.gz package with the libovms_shared.so library and ovms.h header. 
+run `make` command in ovms git main directory.
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+make
+```
+
+## Prepare demo image
+Next enter the directory with the example and build the demo docker with all dependencies and examples that will be named `openvino/model_server-capi`.
+The example image also contains dummy model and config.json required for the applications.
+```bash
+cd demos/capi
+make
```

Comment:
how the user should get the package? We need a step to download the tar file or build from source

Reply:
Building from source is explained above.